popup.js

```diff
@@ -8,6 +8,68 @@ const channelThumbnailInput = document.querySelector('.js-channel-thumbnail-inpu
 const channelNameInput = document.querySelector('.js-channel-name-input')
 const thumbnailInput = document.querySelector('.js-thumbnail-input')
 const errorMessageSpan = document.querySelector("#extErrorMessage")
+const darkModeBtn = document.querySelector('.darkmode-button')
```

Comment:
Here you can for example add js- as prefix for your classname. 
I have for habit to avoid mixing classes that are done for styling purpose and classes that will be called in javascript. 
If you prefix the js classes by .js-, when you look at the html and wants to make some change with the style, you know that theses classes must not be deleted because they are used in script places. 
So maybe your button could have a normal class for styling and another .js- for script purpose. Is it clear?

Reply:
I get it, very clear !
From now I'll do the same :-)

Reply:
If you understood my point here, it's cool. 
You don't have to do this all the time. But for Vanilla JS project I like to do this.
In React, or Vue, or Angular, it is less required